content/blog/explore-the-hpe-greenlake-for-compute-ops-management-rest-api-using-python-and-powershell.md

```diff
@@ -0,0 +1,242 @@
+---
+title: Explore the HPE GreenLake for Compute Ops Management REST API using
+  Python and PowerShell
+date: 2022-12-01T12:28:58.682Z
+author: Vincent Berger & Frederic Passeron
+authorimage: /img/vb-fp192.png
+disable: false
+tags:
+  - "hpe-greenlake "
+  - "hpe-greenlake-for-compute-ops-management "
+  - "developer "
+  - devops
+---
+HPE GreenLake for Compute Ops Management automates and transforms complex and time-consuming compute management operations into a simplified experience across edge-to-cloud. Whether you are an IT OPS or a DEV OPS engineer, you know that automation is the key to success. And today’s automation relies heavily on APIs and how one can interact easily with them. So, let us show you how to leverage the API provided so you, too, can take advantage of what HPE GreenLake for Compute Ops Management can provide.
+
+![figure1](/img/greenlake-com-ops-api-curl1.png)
+
+# **HPE GreenLake for Compute Ops Management REST API**
+
+HPE GreenLake for Compute Ops Management provides a Restful API to customers who want to manage their devices programmatically or through a command line interface. The API enables customers to invoke operations or tasks such as list devices, see device details, device health, manage a device's firmware and much more.
+
+To learn more about HPE GreenLake for Compute Ops Management API methods and resources, you can refer to [the API reference documentation](https://developer.greenlake.hpe.com/docs/greenlake/services/compute-ops/public/openapi/compute-ops-latest/overview/).
+
+Here are some of the operations you can do with the HPE GreenLake for Compute Ops Management API:
+
+* Obtain the list of servers in your account 
+* Obtain the list of firmware bundles
+* Obtain the list of job templates
+* Perform a firmware update
+* Run a carbon emissions report
+* Power on/off/restart a server
+
+![figure2](/img/greenlake-com-ops-api-pyt-pow9.png)
+
+I﻿n a previous [blog](https://developer.hpe.com/blog/explore-the-hpe-greenlake-for-compute-ops-management-rest-api-using-curl-and-postman), we covered some of basics of this API using cURL or Postman. We would like to share with you today a few examples of REST API calls that can be made through simple Python and PowerShell codes examples
```

Comment:
Recommend you rephrase the second sentence to:
In this post, we would like to share with you a few examples of REST API calls that can be made through simple Python and PowerShell code examples.
(note "code" versus "codes", the addition of the period at the end of the sentence, and the change at the beginning - since your today could easily be their tomorrow :-) )